src/wazuh_db/helpers/wdb_agents_helpers.c

```diff
@@ -43,6 +47,10 @@ int wdb_agents_vuln_cve_insert(int id,
     cJSON_AddStringToObject(data_in, "version", version);
     cJSON_AddStringToObject(data_in, "architecture", architecture);
     cJSON_AddStringToObject(data_in, "cve", cve);
+    cJSON_AddStringToObject(data_in, "reference", reference);
```

Comment:
We have to make that **wdb_agents_vuln_cve_insert()** returns a value to the caller to know the status of the insert.
Probably, we should make that the methods in WazuhDB return a json with information of the action on the DB, in order to respond what was done.

Reply:
+1 ... the caller should know in some ways about the error codes.